CHANGELOG.md

```diff
@@ -3,12 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 but version numbers for this repo and a little complicated because they track
-both the Rust version and our own changes no top; consule [`./README.md`] for details.
+both the Rust version and our own changes no top; consult [`./README.md`] for details.
 
 Every commit to `main` in this project creates a new release and hence must have
 a new version number. Fill in an appropriate changelog entry in this file to
 get CI passing and enable the changes to land on `main`.
 
+## 1.63-2.0
```

Comment:
```suggestion
## 1.63-1.1
```

IIUC this is a purely additive change, so we don't need to rev the major version number.